sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/RowSerializerPool.scala

```diff
@@ -1,107 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-package com.azure.cosmos.spark
-
-import com.azure.cosmos.spark.diagnostics.BasicLoggingTrait
-import org.apache.spark.sql.Row
-import org.apache.spark.sql.catalyst.encoders.{ExpressionEncoder, RowEncoder}
```

Comment:
Let's discuss offline what exactly broke here - we should try to keep as mcuh code as possible in a base class RowSerializerPoolBase in this project - and only what really is different now in Spark 3.5 vs. 3.1-3.4 in a derived class in the Spark-specific package